File: app/src/main/java/com/example/acdat_pizzeria/modelo/enums/Tamano.java
package com.example.acdat_pizzeria.modelo.enums;

import java.io.Serializable;

public enum Tamano implements Serializable {
    INDIVIDUAL(5.5, "Individual"),
    MEDIANA(7.5, "Mediana"),
    FAMILIAR(10.0, "Familiar");

    private Double precioInicial;
    private String nombre;

    Tamano(Double precioInicial, String nombre) {
        this.precioInicial = precioInicial;
        this.nombre = nombre;
    }

    public Double getPrecioInicial() {
        return precioInicial;
    }

    public void setPrecioInicial(Double precioInicial) {
        this.precioInicial = precioInicial;
    }

    public String getNombre() {
        return nombre;
    }
}

File: app/src/main/java/com/example/acdat_pizzeria/modelo/enums/kotlin/Tamano.kt
package com.example.acdat_pizzeria.modelo.enums.kotlin

import java.io.Serializable

enum class Tamano(private var precioInicial: Double, private val nombre: String) : Serializable {
    INDIVIDUAL(5.5, "Individual"), MEDIANA(7.5, "Mediana"),
    FAMILIAR(10.0, "Familiar");

    open fun getPrecioInicial(): Double {
        return precioInicial
    }

    open fun setPrecioInicial(precioInicial: Double) {
        this.precioInicial = precioInicial
    }

    open fun getNombre(): String {
        return nombre
    }
}